### PR TITLE
fixed import on google_tags_tag_binding

### PR DESCRIPTION
**Pull request by edwardmedia**
Fixes https://github.com/hashicorp/terraform-provider-google/issues/15944

**Release Note Template for Downstream PRs (will be copied)**

```release-note:bug
tags: fixed import failure on `google_tags_tag_binding`
```

### CODE REVIEW

File: mmv1/templates/terraform/custom_import/tags_tag_binding.go.erb [hunk 1]
```diff
@@ -8,7 +8,12 @@
         return nil, err
     }
 
-    name := d.Get("name").(string)
-    d.SetId(name)
```
[the remark this answers is not in the document]
Comment (edwardmedia) [reply mmv1/templates/terraform/custom_import/tags_tag_binding.go.erb:12]: I don't think so. `parent` is the key for import

[the remark this answers is not in the document]
Comment (edwardmedia) [reply mmv1/templates/terraform/custom_import/tags_tag_binding.go.erb:12]: why do you think it may need?

[the remark this answers is not in the document]
Comment (SarahFrench) [reply mmv1/templates/terraform/custom_import/tags_tag_binding.go.erb:12]: Following on from our DM conversation, [here's an explanation of why d.SetId is needed](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/helper/schema#ResourceData.SetId)

> SetId sets the ID of the resource. If the value is blank, then the resource is destroyed.

**Review by SarahFrench — APPROVED** [review]
LGTM

**Pull request merged**